packages/ui/src/components/Avatar/Avatar.stories.tsx

```diff
@@ -1,38 +1,38 @@
 import React from 'react'
 import { IconAirplay } from '../Icon/icons/IconAirplay'
 
-import Avatar from './Avatar'
+import Avatar, { AvatarProps } from './Avatar'
 
 export default {
   title: 'General/Avatar',
   component: Avatar,
 }
 
-export const Image = (args: any) => {
```

Comment:
same as other comment, should be kitchen sink style

Reply:
leaving for now